README.md

```diff
@@ -1,13 +1,27 @@
 # enclave-cc
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fconfidential-containers%2Fenclave-cc.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fconfidential-containers%2Fenclave-cc?ref=badge_shield)
+## Introduction
 
+  Confidential containers are the product of the combination of confidential computing technology and cloud-native technology. Confidential containers are hardware-based (HW-TEE) for resource isolation, data encryption, and remote attestation, and can prevent theft and tampering of data in the execution environment by cloud service providers and any high-privileged third parties, thus effectively securing user data and assets.
 
-Process-based Confidential Container Runtime
+  Confidential containers have the following characteristics: they provide confidentiality, integrity and security protection for data, especially for runtime data; they provide full-link security for the deployment and operation of sensitive workloads; they provide a way to prove that the environment in which confidential workloads are currently running is real and trustworthy; tenants maintain almost the same experience as using ordinary containers, and deploy sensitive applications without operation and maintenance costs.
+
+  The Confidential Containers project provides a way to protect cloud native applications in HW-TEE without additioanl modification to the container image during deployment. HW-TEE-based confidential container isolation can take two forms: process and virtual machine based isolation.
+
+  Enclave-cc provides process-based confidential container solution leveraging Intel SGX. The term enclave is often associated with process-based isolation. 
+
+## Motivation
+
+  Enclave-CC enables process-based isolation. Process-based isolation is beneficial by drawing the isolation boundary exactly around each container process. This reduces the Trusted Computing Base, or the amount of the system your application depends on for security.
+
+  Consistent with the VM-based isolation design, the enclave-cc approach does not measure and attest to the workload (user application). Instead it uses generic enclaves to isolate user's application from the rest of the system. Measurement and attestation of the empty generic enclaves is used in building trust to deliver the application securely.
```

Comment:
please change `is` in `Measurement and attestation of the empty generic enclaves is` to `are `